dashboard/app/helpers/application_helper.rb

```diff
@@ -7,7 +7,6 @@ module ApplicationHelper
   include LocaleHelper
   include ScriptLevelsHelper
   include ViewOptionsHelper
-  include LevelsHelper
```

Comment:
This was removed to avoid a circular dependency. In order for LevelsHelperTests to test LevelsHelper, LevelsHelper needs to include ApplicationHelper to access the browser method. Once that happens, we can't have ApplicationHelper include LevelsHelper. I discussed this with Laurel yesterday and arrived at this solution.